src/main/java/Task.java

```diff
@@ -0,0 +1,27 @@
+public class Task {
+    private String title;
+    private boolean completed = false;
+
+    public Task() {
+        this("null");
```

Comment:
i think a more intuitive default name could be better like "Untitled Task", unless your intention is to use this to indicate nullish...?